extensions/amp-list/amp-list.md

```diff
@@ -280,9 +280,47 @@ See below for a full example,
 </amp-list>
 ```
 
+### Using amp-script as a data source
+
+You may use an exported `<amp-script>` function as the data source for `<amp-list>`. This enables you to flexibly combine and transform server responses before handoff to `<amp-list>`. The required format is the `<amp-script>` ID and the function name separated by a `.`, e.g. `amp-script:id.functionName`.
+
+See below for an example:
+
+```html
+<!--
+  See the [amp-script](https://amp.dev/documentation/components/amp-script/) documentation to setup the component and export your function>
```

Comment:
need to figure out the correct format for specifying links in html comments for our docs